tests/testApproximate.py

```diff
@@ -53,7 +53,8 @@ def makeRamp(self, binsize=1):
         rampCoeffs = (1000, 1, 1)
         for i in range(ramp.getHeight()):
             for j in range(ramp.getWidth()):
-                ramp.set(j, i, (rampCoeffs[0] + rampCoeffs[1]*x[j] + rampCoeffs[2]*y[i], 0x0, var))
+                ramp.set(
+                    j, i, (rampCoeffs[0] + rampCoeffs[1]*x[j] + rampCoeffs[2]*y[i], 0x0, var))
```

Comment:
The question is a bit moot, because I've already made all the formatting changes I intend to.

As far as I can tell, though, autopep8 only intervenes if it sees a PEP8 violation. It does not, for example, remove line breaks that were added by hand.